### PR TITLE
Bookstore: initialize project with components

### PR DESCRIPTION
**Pull request by TesheMaximillan**
# In this Milestone I followed the given requirements📌✅♻️

* Initialize React app.
* Reusable components are created.
* Add React router and set two `<Route>` and `<Link>` for the app navigation
* Books page is the default view
* Display the list of books with the Remove button
* Create a component book for displaying a single book
* A form for adding books is created
* Categories should have a button "Check status" only

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,7 +1,20 @@
...
+import NotMatch from './pages/NoMatch';
 
 function App() {
-  return <div className="app">App</div>;
+  return (
+    <Router>
+      <NavBar />
+      <Routes>
+        <Route path="/" element={<Books />} />
+        <Route path="*" element={<NotMatch />} />
```
Comment (FrederickMih) [inline src/App.js]: - [ ] I believe this is an optional component for any route that does not match right?.  You will also require a `<Route>` for the `Categories` component and should display only a button with the text "Check status" as the requirement says, then you can optionally add this component(NotMatch) for any other route which is also important :+1:. It's vital for us to always stick to project requirements before including additional features.

**Review by FrederickMih — CHANGES_REQUESTED** [review]
Hi @TesheMaximillan , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

# To Highlights

-  Linters checks are all passing with green check marks :heavy_check_mark: 
- PR has a title and summary :heavy_check_mark: 
- Git/GitHub workflow has been followed :+1: 


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by TesheMaximillan** [comment]
> To follow the requirement:

* I removed `NoMatch` Component
* Add a router for Categories as your comment

**Review by nicupop729 — APPROVED** [review]
Hi @TesheMaximillan, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
 - [X] Correct git flow practice :heavy_check_mark:
 - [X] Good commit messages :heavy_check_mark:
 - [X] Professional README :heavy_check_mark:
 - [X] Linters are passing :heavy_check_mark:
 - [X] Clean codebase :heavy_check_mark:

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**